vaje.tex

```diff
@@ -21,6 +21,13 @@
 % Naprednejši ukazi
 \usepackage{xparse}
 
+% Izbor pisave
+\usepackage{mathpazo}
+\usepackage[scaled=0.95]{helvet}
+\usepackage{courier}
+\linespread{1.05} % Palatino looks better with this
```

Comment:
komentarji naj bodo v slovenščini

Reply:
Meni se to ne zdi potrebno, ampak ok, če tako pravi slovenska zakonodaja. Bom popravil.